index.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="styles/styles.css">
+  <title>Rekupe</title>
+</head>
+<body>
+
+<!--HEADER-->
+<header class="header-outer-container" id="header">
+
+  <div class="header-inner-container">
+    <nav class="header-nav">
+      <ul>
+        <li><a href="#studies">Case Studies</a></li>
+        <li><a href="#services">Services</a></li>
+        <li><a href="#contact">Contact Us</a></li>
+      </ul>
+    </nav>
+
+    <div class="header-content">
+      <h1>Your website is an investment in the success of your business.</h1>
+      <p>We are here to help you succeed.</p>
+      <button type="button">Let's work together</button>
+    </div>
+  </div>
+
+  <div class="header-wave-icon"></div>
+
+</header>
+
+<!--MAIN-->
+
+<main>
+
+<!--INFO SECTION  -->
+
+  <section id="info" class="info-outer-container " >
+
+    <div class="info-inner-container">
+      <h5>Rekupe is a digital agency in Los Angeles.</h5>
+      <h5>We use strategic UX, SEO optimization and beautiful visual designs to help your business grow.</h5>
+      <h5>We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.</h5>
+      <p>Have a question? <a href="#contact">Click here</a> to contact us!</p>
+    </div>
+
+  </section>
+
+<!--STUDIES SECTION-->
+
+  <section id="studies" class="studies-outer-container" >
+    <div class="studies-inner-container">
+
+    <article class="studies-article">
+      <div class="studies-article-details">
+        <h3>Rolling Stone</h3>
+        <p>You know Rolling Stone. It’s the definitive voice in music, politics and culture. It’s a global icon.</p>
+        <a class="studies-article-details-link" href="dummy.html">View project</a>
+      </div>
+
+      <span class="studies-article-image"><img src="/assets/work-image-01.jpeg" alt="Image Rolling Stone"/></span>
```

Comment:
Ik moest het relatieve pad aanpassen naar
```suggestion
      <span class="studies-article-image"><img src="./assets/work-image-01.jpeg" alt="Image Rolling Stone"/></span>
```
om het plaatje te kunnen zien. Dit geldt trouwens voor alle afbeeldingen in deze html.

Reply:
overal aangepast